types/light_info.go

```diff
@@ -0,0 +1,12 @@
+package types
```

Comment:
Could this live in `light.go`? That file already has light-client related types in it, and it's not clear why this one is separate.

Reply:
The only reason I put it in types is to avoid importing the light client package into rpc/coretypes. It is already importing types to get NodeInfo and we have LightBlock for example defined in the types package as well.

Reply:
Actually @creachadair most of the light client types are in the `types` directory under `types/light.go`

Reply:
> Actually @creachadair most of the light client types are in the `types` directory under `types/light.go`

Yes, that's where I was suggesting this could go.

Reply:
Ah sorry I misread it as the `light` package